README.markdown

```diff
@@ -125,6 +125,10 @@ Make sure you have the file `~/.m2/settings.xml`:
           <username></username>
           <password></password>
         </server>
+        <server>
+          <id>github-project-site</id>
```

Comment:
`github-project-site` is the id that is looked up from the settings by the github site wagon plugin, and `git` is the username that github requires.  So yes, these values should be pasted as is.